spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe QueueItem do  
+  it { should belong_to(:user) }
+  it { should belong_to(:video) }
+
+  describe "#title" do
```

Comment:
#title ... I try to work on the exercise before resorting to the video. I already created the migration with title instead of video_title